### PR TITLE
To Vaporetto v0.5.1

### PR DESCRIPTION
**Pull request by kampersanda**
Vaporetto with `rev = "a8308cd"` has been outdated, and the compilation fails.
https://github.com/daac-tools/vaporetto/search?q=hash%3Aa8308cd&type=commits

This PR updated the version to the latest.

### CODE REVIEW

File: bench/vaporetto-bench/Cargo.toml [hunk 1]
```diff
@@ -2,13 +2,12 @@
 name = "vaporetto-bench"
 version = "0.1.0"
 edition = "2021"
-resolver = "2"
 
 [dependencies]
-vaporetto = { git = "https://github.com/daac-tools/vaporetto.git", rev = "a8308cd", default-features = false, features = ["cache-type-score", "fix-weight-length"] }
+vaporetto = { version = "0.5.1" }
```
Comment (vbkaisetsu) [inline bench/vaporetto-bench/Cargo.toml]: Could you fix the version?
```suggestion
vaporetto = "=0.5.1"
```

File: bench/vaporetto-bench/Cargo.toml [hunk 2]
```diff
@@ -2,13 +2,12 @@
...
 
 [build-dependencies]
-vaporetto = { git = "https://github.com/daac-tools/vaporetto.git", rev = "a8308cd", default-features = false, features = ["std", "cache-type-score", "fix-weight-length", "kytea"] }
+vaporetto = { version = "0.5.1", features = ["kytea"] }
```
Comment (vbkaisetsu) [inline bench/vaporetto-bench/Cargo.toml]: ditto

File: bench/vaporetto-bench/build.rs [hunk 3]
```diff
@@ -3,18 +3,16 @@
...
 fn main() {
-    let mut model_data = Cursor::new(include_bytes!("../../resources/jp-0.4.7-5.mod"));
-    let model = KyteaModel::read(&mut model_data).unwrap();
+    let model_data = Cursor::new(include_bytes!("../../resources/jp-0.4.7-5.mod"));
+    let model = KyteaModel::read(model_data).unwrap();
     let model = Model::try_from(model).unwrap();
-    let predictor = Predictor::new(model, false).unwrap();
     let out = &PathBuf::from(env::var_os("OUT_DIR").unwrap());
     let mut buf = BufWriter::new(File::create(out.join("predictor.bin")).unwrap());
-    let model_data = predictor.serialize_to_vec().unwrap();
```
Comment (vbkaisetsu) [inline bench/vaporetto-bench/build.rs:18]: Could you keep the serialization?

**Review by vbkaisetsu — CHANGES_REQUESTED** [review]
.

File: bench/vaporetto-bench/Cargo.toml (hunk 2 above)
Comment (kampersanda) [reply bench/vaporetto-bench/Cargo.toml]: https://github.com/legalforce-research/tokenizer-speed-bench/pull/1/commits/b3fe081c9786f1b24c1a844ada727a239ea4e4b2

File: bench/vaporetto-bench/build.rs (hunk 3 above)
Comment (kampersanda) [reply bench/vaporetto-bench/build.rs:18]: Oh, sorry, I misunderstood that `Predictor::serialize_to_vec` was gone. Reverted.

https://github.com/legalforce-research/tokenizer-speed-bench/pull/1/commits/b3fe081c9786f1b24c1a844ada727a239ea4e4b2

**Comment by kampersanda** [comment]
@vbkaisetsu 

> Why did you

`rev=a8308cd` has been outdated and fails to compile with the current version.

```
$ cargo build --release
    Updating git repository `https://github.com/daac-tools/vaporetto.git`
error: failed to get `vaporetto` as a dependency of package `vaporetto-bench v0.1.0 (/Users/shunsuke.kanda/Workspace/github.com/legalforce-research/tokenizer-speed-bench/bench/vaporetto-bench)`

Caused by:
  failed to load source for dependency `vaporetto`

Caused by:
  Unable to update https://github.com/daac-tools/vaporetto.git?rev=a8308cd

Caused by:
  revspec 'a8308cd' not found; class=Reference (4); code=NotFound (-3)
```

**Comment by vbkaisetsu** [comment]
> Why did you

Could you ignore the above comment?
I sent the draft by mistake.

**Review by vbkaisetsu — APPROVED** [review]

**Pull request merged**